client/components/container/container-toolbar-directive.js

```diff
@@ -40,6 +40,18 @@ goog.scope(function() {
 
         /** @export {!DashboardService} */
         this.dashboardSvc = dashboardService;
+
+        /**
+         * Prompts the user for confirmation, then deletes the selected container.
+         * @export
+         */
+        this.removeSelectedContainer = function() {
+          if (!window.confirm('Are you sure you want to delete this container?')) {
```

Comment:
If possible, please include information about the deleted object's identity in the message, just to avoid any confusion about which object is getting deleted. That's not necessary if visible highlighting makes it clear.

Also, to be nitpicky, "are you sure" is a yes/no question which doesn't really fit the "OK/cancel" choices of the confirm dialog, but that seems to be a common pattern. I think "The selected items will be permanently deleted: (item description)" would be clearer.